packages/components/form/src/form.ts

```diff
@@ -107,6 +112,11 @@ export class Form<T extends Record<string, unknown> = Record<string, unknown>> e
     }
   }
 
+  override firstUpdated(changes: PropertyValues<this>): void {
+    super.firstUpdated(changes);
+    this.#initialValue = this.#value;
```

Comment:
Should we have 2 places where we set the `#initialValue`?

Reply:
one is for when the value is set, the other for when it isn't set. No idea why, but it doesn't work with just one